vendor/go.etcd.io/etcd/api/v3/etcdserverpb/rpc.proto

```diff
@@ -217,7 +217,7 @@ service Maintenance {
   // It only iterates "key" bucket in backend storage.
   rpc HashKV(HashKVRequest) returns (HashKVResponse) {
       option (google.api.http) = {
-        post: "/v3/maintenance/hash"
+        post: "/v3/maintenance/hashkv"
```

Comment:
@serathius @logicalhan is this a client-side change, or a server-side change, or both? Will a new client linked with this break against an old server that isn't expecting to handle that path?

Reply:
This is a path on server side used by built in grpc-gateway to provide V3 API via HTTP. Not used by Kubernetes as it uses GRPC and not HTTP API.

Reply:
Context https://github.com/etcd-io/etcd/pull/16464

Reply:
glad k8s doesn't use it... does the bug mean an existing client trying to use the /hash endpoint would have failed?

Reply:
cc @ahrtr

Reply:
sounds good, thanks for clarifying